### ISSUE TITLE
The way to download xhamster.com

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#159]
```
What steps will reproduce the problem?
1. xhamster currently does not work
2.
3.

Please someone add xhamster.com. I figured out how it works, here is a little 
bash script. If any dev wants to add it to the source... 

URL=$1
wget -O testfile -o wget_1log $URL
server=$(cat testfile |grep \'srv\'|cut -d \' -f4|cut -d '/' -f3);
file=$(cat testfile |grep \'file\'|cut -d \' -f4);
wget "http://"$server"/flv2/"$file""


```

Original issue reported on code.google.com by `sinnlose...@yahoo.de` on 27 Aug 2010 at 5:18

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git (commit 72f1383).
```

Original comment by `zakflash...@gmail.com` on 15 Oct 2010 at 2:36
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
wget -c "http://"$server"/flv2/"$file""

```

Original comment by `rica...@companhiadesistemas.com.br` on 3 May 2011 at 1:31
